src/main/java/org/kpmp/spatialViewerDataset/SpatialViewerExternalLinkDataset.java

```diff
@@ -5,10 +5,12 @@
 import javax.persistence.Id;
```

Comment:
Are there tests around this class? If so, is the test class named correctly (SpatialViewerExternalLinkDatasetTest)?